### ISSUE TITLE
ui5-datepicker: Screen Reader Support implementation for picker & calendar

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#1279]
**Is your feature request related to a problem? Please describe.**
The ui5-datepicker should be accessible for all users. Currently, the screen reader support is enabled only for the input field. All needed aria attributes and roles should be added also to the picker and calendar inside.

**Describe the solution you'd like**
Check the sap.m.DatePicker screen reader specification (all things related with the picker and calendar part), adapt it to the ui5-datepicker component and implement it.

**Additional context**
The implementation should be checked with an ACC expert.
